server/lib/configuration.js

```diff
@@ -411,6 +411,11 @@ var conf = module.exports = convict({
       default: undefined,
       doc: 'Remote Sentry endpoint',
       format: String
+    },
+    server_errors_dsn: {
+      default: undefined,
```

Comment:
if you want to test this grab the value from https://sentry.prod.mozaws.net/operations/content-express-server-dev/settings/install/node/ and paste it into `local.json`